project-addons/sale_point_programme/template_participations.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0"?>
+<openerp>
+    <data>
+
+        <record id="template_participations" model="email.template">
```

Comment:
Este fichero creo que no es necearftio ya lo tenéis en project-addons/sale_point_programme/views/template_participations.xml ademñas que no estaría migrado a Odoo 11

Reply:
borrado